src/ui/components/Events/ReplayInfo.tsx

```diff
@@ -53,6 +53,12 @@ function ReplayInfo({ setModal }: PropsFromRedux) {
           <MaterialIcon iconSize="xl">{icon}</MaterialIcon>
           <div>{summary}</div>
         </Row>
+        <Row>
+          <MaterialIcon iconSize="xl">language</MaterialIcon>
+          <div className="overflow-hidden overflow-ellipsis whitespace-pre" title={recording.url}>
+            {recording.url}
```

Comment:
we'll probably want to format it so it only shows the query params on hover, but this is a good first step

Reply:
Fixed!